gateways/core/calendar/start.go

```diff
@@ -43,15 +47,22 @@ func (ce *CalendarConfigExecutor) StartConfig(config *gateways.ConfigContext) {
 
 	for {
 		select {
-		case <-config.StartChan:
-			ce.GatewayConfig.Log.Info().Str("config-name", config.Data.Src).Msg("configuration is running")
-			config.Active = true
+		case _, ok := <-config.StartChan:
+			if ok {
+				ce.GatewayConfig.Log.Info().Str("config-name", config.Data.Src).Msg("configuration is running")
+				config.Active = true
+			}
```

Comment:
do you need an `else` condition if there is a failure receiving from the `StartChan`?

Reply:
If there was any failure before receiving start signal on  `StartChan` configuration state will be marked as `Error`.